turbopack/crates/turbopack-ecmascript/src/lib.rs

```diff
@@ -255,6 +250,31 @@ pub struct EcmascriptModuleAsset {
     last_successful_parse: turbo_tasks::State<Option<ReadRef<ParseResult>>>,
 }
 
+#[turbo_tasks::value_trait]
+pub trait Parsable {
```

Comment:
Maybe for consistency call it `EcmascriptParsable` and `EcmascriptAnalyzable` similar to `EcmascriptChunkPlaceable`?

Reply:
Done